src/index.js

```diff
@@ -1 +1,41 @@
-console.log('Hello');
\ No newline at end of file
+const fs = require('fs');
+const path = require('path');
+const csv = require('csvtojson');
+
+const main = async () => {
+    const txtDir = path.join(__dirname, '../', 'txt');
+    const txtFilePath = path.join(__dirname, '../', 'txt/txt-result.txt');
+        if (!fs.existsSync(txtDir)) {
+            try {
+                await fs.promises.mkdir(txtDir);
+            } catch(err) {
+                return console.log('Error: ', err.message);
+            }
+        }
+        csv()
+        .fromFile(path.join(__dirname, '../', 'csv/csv-sample.csv'))
```

Comment:
Following the break down suggestion above, callbacks extracted to their own named functions would also make a positive impact on overall readability. They could also be kept inside those high-abstraction-level function/steps from the above comment, just to keep the related logic grouped together.

Reply:
Moved the callbacks to the same utils folder as I've tried to make them reusable functions.